### ISSUE TITLE
Sometimes I get de error 'Missing image layout'

### ISSUE THREAD

**Issue by MrAndre96** [Bionus/imgbrd-grabber#1402]
**Bug description**

Sometimes when I am loading a page I get the error 'Missing image layout' inside the log tab.

**Expected behavior**

Have it work without those errors

**Screenshots**

[https://i.imgur.com/kSRc5vI.png](https://i.imgur.com/kSRc5vI.png)

**System information**

Windows 10 with the latest nightly version

**Comment by Bionus** [comment]
Indeed, it seems to happen because Grabber fails to properly cancel loading of pages with a page delay setting. I'll add more information in the log line and fix the cancellation so this log shouldn't show up anymore. :+1:

**Comment by MrAndre96** [comment]
@Bionus When you go fast to the next page a couple times behind each other (possibly when sankaku complex has a delay for the pages and images enabled) I still get the 'Missing image layout' error. It also seems to be the case that the error of my other topic is also because of this since I see those warning appear when I tested this out.

[https://i.imgur.com/XXA5C9t.png](https://i.imgur.com/XXA5C9t.png)

**Comment by Bionus** [comment]
In your screenshot, it seems to be loading full-size images instead of thumbnails: [https://files.yande.re/image/e5da2ba5100a57daabecdf730281c5c3/yande.re 393993 animal_ears azuki breast_hold coconut heterochromia naked nekopara sayori tail wallpaper.jpg](https://files.yande.re/image/e5da2ba5100a57daabecdf730281c5c3/yande.re 393993 animal_ears azuki breast_hold coconut heterochromia naked nekopara sayori tail wallpaper.jpg) (**NSFW**) instead of https://assets.yande.re/data/preview/e5/da/e5da2ba5100a57daabecdf730281c5c3.jpg (**NSFW**). Did you change any setting that might have caused that? Otherwise, that might be linked to this bug 🤔
